Sources/web3swift/HookedFunctions/Web3+BrowserFunctions.swift

```diff
@@ -74,15 +74,18 @@ extension web3.BrowserFunctions {
     
     
     public func sendTransaction(_ transactionJSON: [String: Any], password: String = "web3swift") -> [String:Any]? {
-        guard let transaction = EthereumTransaction.fromJSON(transactionJSON) else {return nil}
-        guard let options = TransactionOptions.fromJSON(transactionJSON) else {return nil}
-        var transactionOptions = TransactionOptions()
-        transactionOptions.from = options.from
-        transactionOptions.to = options.to
-        transactionOptions.value = options.value != nil ? options.value! : BigUInt(0)
-        transactionOptions.gasLimit = options.gasLimit != nil ? options.gasLimit! : .automatic
-        transactionOptions.gasPrice = options.gasPrice != nil ? options.gasPrice! : .automatic
-        return self.sendTransaction(transaction, transactionOptions: transactionOptions, password: password)
+        do {
+          let jsonData: Data = try JSONSerialization.data(withJSONObject: transactionJSON, options: [])
+          let transaction: EthereumTransaction = try JSONDecoder().decode(EthereumTransaction.self, from: jsonData)
+          let options: TransactionOptions = try JSONDecoder().decode(TransactionOptions.self, from: jsonData)
+          var transactionOptions = TransactionOptions()
+          transactionOptions.from = options.from
+          transactionOptions.to = options.to
+          transactionOptions.value = options.value != nil ? options.value! : BigUInt(0)
```

Comment:
Do you mind if we touch this block, then change this code to a more readable one with nil coalescing? And for all the others

From:
`transactionOptions.value = options.value != nil ? options.value! : BigUInt(0)`

To:
`transactionOptions.value = options.value ?? BigUInt(0)`

Reply:
yep, I should have caught that